bodhi/server/templates/override.html

```diff
@@ -160,9 +160,9 @@ <h2 class="pull-left">New Override</span>
   </div>
 </div>
 </div>
-<script src="${request.static_url('bodhi:server/static/js/override_form.js')}"></script>
-<script src="${request.static_url('bodhi:server/static/datepicker/js/bootstrap-datepicker.js')}"></script>
-<link href="${request.static_url('bodhi:server/static/datepicker/css/datepicker.css')}" rel="stylesheet" />
+<script src="${request.static_url('bodhi:server/static/js/override_form.js')}?bodhi=${self.util.version()}"></script>
+<script src="${request.static_url('bodhi:server/static/datepicker/js/bootstrap-datepicker.js')}?bodhi=${self.util.version()}"></script>
+<link href="${request.static_url('bodhi:server/static/datepicker/css/datepicker.css')}?bodhi=${self.util.version()}" rel="stylesheet" />
```

Comment:
This seems like a fine way to do it, but I have an idea of a way to do it that will be less lines of change and will also ensure that it stays fixed: We could modify ```request.static_url()``` to look for parameters that start with ```bodhi:server/static``` and then automatically append the version.

What do you think?